Essentials/src/messages.properties

```diff
@@ -89,6 +89,7 @@ defaultBanReason=The Ban Hammer has spoken\!
 deleteFileError=Could not delete file\: {0}
 deleteHome=\u00a76Home\u00a7c {0} \u00a76has been removed.
 deleteJail=\u00a76Jail\u00a7c {0} \u00a76has been removed.
+deleteKit=\u00a72The\u00a7c {0}\u00a72 kit successfully deleted.
```

Comment:
This is still not consistent with the other messages - should read `Kit {kit} has been removed.` with the same colours as the messages right above.